src/server/services/procedures/FBICrimeData/FBICrimeData.js

```diff
@@ -91,7 +91,7 @@ const OFFENSE_Data_Controller_OPTIONS = {
 
 types.defineType({
     name: 'OffenseData',
-    description: 'Type of information to get offense.',
+    description: 'NIBRS Offense Count Endpoint',
```

Comment:
So an endpoint is sort of a backend topic that the students don't need to know about. But more importantly, this is just a list of valid inputs to give the endpoint (api), rather than being an endpoint itself. You just need to describe what this type is used for (e.g., what the values represent in terms of the data).

(There were other instances of "endpoint" as well).